Mage.Tests/src/test/java/org/mage/test/player/TestPlayer.java

```diff
@@ -393,7 +393,8 @@ public boolean hasAbilityTargetNameOrAlias(Game game, Ability ability, String na
             String searchObject = nameOrAlias.substring("Cast ".length());
             if (searchObject.contains(" with ")
                     || searchObject.contains(" using ")
-                    || searchObject.contains("fused ")) {
+                    || searchObject.contains("fused ")
+                    || searchObject.contains("plotted")) {
```

Comment:
Smells bad. Where are foretell, suspend and other same abilities? Why you need that hack for a plot?

Reply:
Well the one that does not make much sense is Foretell:
![image](https://github.com/magefree/mage/assets/34709007/01269302-fcc7-43b5-a417-9622e11ee15b)

The extra care to have the spell ability starting with "Cast plotted: " is why I added that contains to write the tests.

Suspend looks like
![image](https://github.com/magefree/mage/assets/34709007/771b0a5a-c06e-4d0e-a258-80b2967c3712)